ra_aid/prompts.py

```
@@ -620,9 +640,247 @@
+# New agentic chat prompt for interactive mode
+CHAT_PROMPT = """Working Directory: {working_directory}
+Current Date: {current_date}
+Project Info:
+{project_info}
+
+Agentic Chat Mode Instructions:
+
+Overview:
+    In this mode, you will function as an interactive agent that relies on direct human input to guide your actions.
+    You must always begin by using ask_human to request an initial task or set of instructions from the user.
+    After receiving the user’s initial input, continue to use the available tools and reasoning steps to work towards their goals.
```

Ruff (0.8.2)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)